gui/qt/main_window.py

```diff
@@ -1428,6 +1447,9 @@ def do_send(self, preview = False):
         if (fee < (tx.estimated_size())):
             msg.append(_('Warning') + ': ' + _("You're using a fee less than 1000 sats/kb.  It may take a very long time to confirm."))
 
+        if ((self.message_opreturn_e.text() is not None) and (self.message_opreturn_e.text() !="")):
```

Comment:
Can simplify to:

if not self.message_opreturn_e.text():

Reply:
Also these parentheses are superfluous and negatively impact readability.

And binds after 'is not' in the order of operations.

Reply:
also please rewrite this as:

`if self.message_opreturn_e.text():`